app/src/main/java/fr/free/nrw/commons/upload/mediaDetails/UploadMediaDetailFragment.java

```diff
@@ -458,6 +479,16 @@ public void showExternalMap(final UploadItem uploadItem) {
         goToLocationPickerActivity(uploadItem);
     }
 
+    @Override
```

Comment:
javadoc

Reply:
Even for overridden methods, it is good to write javadoc (which can be taken from the parent class if applicable). Every written javadoc is then read by hundreds of people, so having the information wherever it is needed is important.